pkg/oc/cli/cmd/rsync/copyrsync.go

```diff
@@ -111,6 +110,21 @@ func (r *rsyncStrategy) Validate() error {
 	return nil
 }
 
+// rsyncEscapeCsommand wraps each element of the command in double quotes
+// if it contains any of the following: single quote, double quote, backslash, or space
+// It also replaces every pre-existing double quote character in the element with a pair.
+// Example: " -> ""
+func rsyncEscapeCommand(command []string) []string {
+	for key := range command {
+		needsQuoted := strings.ContainsAny(command[key], `'" \`)
+		if needsQuoted {
+			command[key] = strings.Replace(command[key], `"`, `""`, -1)
```

Comment:
ok but this is escaping all the parts of the command invocation, right?  e.g.:

["/usr/bin/oc","rsync","podname","/foo/bar"]

so they shouldn't all be escaped that way, just the parts that are the path arguments to rsync, right?